src/ucp/rndv/rndv_get.c

```diff
@@ -329,15 +330,12 @@ ucp_proto_rndv_get_mtype_probe(const ucp_proto_init_params_t *init_params)
             continue;
         }
 
-        status = ucp_mm_get_alloc_md_index(context, frag_mem_info.type,
-                                           &dummy_md_id,
+        status = ucp_mm_get_alloc_md_index(context, frag_mem_info, &dummy_md_id,
```

Comment:
according to how ucp_mm_get_alloc_md_index function is used, seems we would want to pass mem_type and sys_dev to it (as separate parameters), and it shall return ucp_memory_info_t and md_index.
for example, in rndv_get, we can introduce a local var for mem_type iteration, and pass sys_dev directly from init_params->select_param->sys_dev